Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -811,8 +813,6 @@ static bool isStateValid(const auto& source, const std::optional<GPUOrigin2D>& o
     );
 }
 
-// FIXME: https://bugs.webkit.org/show_bug.cgi?id=263692 - this code should be removed, it is to unblock
```

Comment:
I prefer to keep the WCP path for when the data is not accelerated or otherwise already in an IOSurface